test/integ/dataplaneapi/conftest.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import pytest
+import boto3
+import requests
+import logging
+import re
+import os
+
+from requests_aws4auth import AWS4Auth
+
+
+# Fixture for retrieving env variables
+
+@pytest.fixture(scope='session')
+def testing_env_variables():
+    print('Setting variables for tests')
+    try:
+        test_env_vars = {
+            'MEDIA_PATH': os.environ['TEST_MEDIA_PATH'],
```

Comment:
Where are all these environment variables set?

Reply:
No, these are automatically set in the run_integ.sh script that invokes the test files.

Reply:
NM. See it in the run test shell script.